recipe/meta.yaml

```diff
@@ -108,10 +86,22 @@ outputs:
 about:
   home: https://github.com/microsoft/onnxruntime/
   summary: cross-platform, high performance ML inferencing and training accelerator
-  license: MIT
+  license: MIT  # BSL-1.0  # mp11
```

Comment:
It seems that I can't put a list of license names here. The problem arises because the vendored version of boost-mp11 does not bring its own license file.

Reply:
You can use the AND operator here, see https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/